streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorNode.java

```diff
@@ -71,4 +71,20 @@ public void process(K key, V value) {
     public void close() {
         processor.close();
     }
+
+    /**
+     *
+     * @return
+     */
+    public String toString() {
+        String processorNodeString = "ProcessorNode[name=" + name;
+        if (stateStores != null && !stateStores.isEmpty()) {
+            processorNodeString += "," + "StateStores[";
+            for (String store : stateStores) {
+                processorNodeString += "store=" + store + ",";
```

Comment:
This will print as `StateStores[store=store1, store=store2, store=store3,]`, maybe just `StateStores[store1, store2, store3]` is fine.